plugins/inputs/aerospike/aerospike.go

```diff
@@ -121,6 +121,8 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 func parseValue(v string) interface{} {
 	if parsed, err := strconv.ParseInt(v, 10, 64); err == nil {
 		return parsed
+	} else if parsed, err := strconv.ParseUint(v, 10, 64); err == nil {
+		return -int64(parsed)
```

Comment:
also telegraf might not want to send a string for a number that is too large. I don't think it's aerospike's fault to be sending uints. We might want to just drop metrics that are beyond the maximum int64 (or simply cap the the metric)